docs/StardustDocs/topics/read.md

```diff
@@ -60,7 +71,7 @@ C: Double
 D: Boolean?
 ```
 
-[`DataFrame`](DataFrame.md) will try to parse columns as JSON, so when reading following table with JSON object in column D:
+[`DataFrame`](DataFrame.md) tries to parse columns as JSON, so when reading the following table with JSON object in column D:
```

Comment:
I'd like stylistic changes, looks like we need common recommendations like "use Present Simple instead of Future Simple"